website/versioned_docs/version-2.0.0-alpha.43/deployment.md

```diff
@@ -146,3 +146,35 @@ Render offers [free static site hosting](https://render.com/docs/static-sites) w
    | **Publish Directory** | `build`       |
 
 That's it! Your app will be live on your Render URL as soon as the build finishes.
+
+## Automating Deployments Using Continuous Integration
```

Comment:
@mohamedsgap please remove change in this file. Docs for version 43 has already been released, so we cannot and should not add anything new to this version.